packages/web-components/src/components/skip-to-content/__stories__/README.stories.mdx

```diff
@@ -0,0 +1,26 @@
+import { Preview, Props, Description, Story } from '@storybook/addon-docs/blocks';
+import contributing from '../../../../../../docs/contributing-license.md';
+
+# Skip to content
+
+> The skip to content component works as an naivgation link that help users move around a page with keyboard.
+
+> 💡 Check our
+> [CodeSandbox](https://githubbox.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/web-components/examples/codesandbox/components/skip-to-content)
+> example implementation.
+
+[![Edit @carbon/ibmdotcom-web-components](https://codesandbox.io/static/img/play-codesandbox.svg)](https://githubbox.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/web-components/examples/codesandbox/components/skip-to-content)
+
+## Getting Started
+
+### JS
+
+```javascript
+import '@carbon/ibmdotcom-web-components/es/components/skip-to-content/skip-to-content';
+```
+
+### HTML
+
+```html
+<dds-skip-to-content></dds-skip-to-content>
```

Comment:
Thank you @asudoh ! For the README, can you add documentation on the `href` property?

Reply:
Thank you for the feedback @jeffchew! After thinking a while; Now trying to implement this in `carbon-web-components`: carbon-design-system/carbon-web-components#486 - The doc includes the prop description. Will update this PR once carbon-design-system/carbon-web-components#486 is merged and released.

Reply:
Made an update to use `<bx-skip-to-content>` from `carbon-web-components`.